res/controllers/Numark-Mixtrack-Pro-FX-scripts.js

```diff
@@ -0,0 +1,788 @@
+// dim all lights when inactive instead of turning them off
+components.Button.prototype.off = 0x01;
```

Comment:
you can do this whenever you want before executing the init function (done by mixxx). I'd put this after the user-configurable variables.